flake.nix

```diff
@@ -2,20 +2,17 @@
   description = "Ply - A helper library for working with compiled, parameterized Plutus Scripts";
 
   inputs = rec {
-    haskell-nix.url = "github:mlabs-haskell/haskell.nix";
-
+    haskell-nix.url = "github:input-output-hk/haskell.nix";
     nixpkgs.follows = "haskell-nix/nixpkgs-unstable";
+    extra-hackage.url = "github:mlabs-haskell/haskell-nix-extra-hackage?ref=ee50d7eb739819efdb27bda9f444e007c12e9833";
+    extra-hackage.inputs.haskell-nix.follows = "haskell-nix";
+    extra-hackage.inputs.nixpkgs.follows = "nixpkgs";
 
     iohk-nix.url = "github:input-output-hk/iohk-nix";
     iohk-nix.flake = false; # Bad Nix code
 
     plutarch = {
-      url = "github:Plutonomicon/plutarch?ref=staging";
-      inputs = {
-        haskell-nix.follows = "haskell-nix";
-        nixpkgs.follows = "nixpkgs";
-        cardano-base = cardano-base;
-      };
+      url = "github:Plutonomicon/plutarch-plutus/e23f2d9ecad8482e9971bd6ec1c772f9693014ce";
```

Comment:
@TotallyNotChase should we keep the rev or just say staging?

Reply:
I'll try with staging actually